src/services/participantsService.js

```diff
@@ -158,7 +158,7 @@ const leaveConversation = async function(token) {
  * @param {string} token The conversation token;
  */
 const leaveConversationSync = function(token) {
-	axios.delete(generateOcsUrl('apps/spreed/api/v3/room', 2) + token + '/participants/active')
+	axios.delete(generateOcsUrl('apps/spreed/api/v4', 2) + `room/${token}/participants/active`)
```

Comment:
this would not escape the token arg, but it's ok

sadly generateOcsUrl doesn't seem to accept params, but generateUrl does: https://github.com/nextcloud/nextcloud-router/blob/master/lib/index.ts#L53

I've raised it here for later: https://github.com/nextcloud/nextcloud-router/issues/194

since the token is known to be safe, ok to keep

Reply:
Oh, I didn't know this as all APIs I use are OCS. sounds like a good enhancement

Reply:
Yeah, I think it used to be that way when it was still part of server/core and didn't get ported that way.